hack/arktos-cni.rc

```diff
@@ -66,6 +66,21 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+install_cni_mizar() {
```

Comment:
Shouldn't the yaml deployment do this?

Reply:
Yes, Unless the mizar-daemon and mizar-operator come in running state, the cluster deployment waits for a very long time with ```Waiting for node ready at api server``` message.
So, we added the above code.

Reply:
What's the reason the daemon and operator take a long time to come up to running state? It happens pretty quickly in AWS. It should be the same behavior in single cluster as well.

Reply:
This issue was resolved without any explanation.

Reply:
The operator pod comes in running early, but the daemon pod has initContainer which runs the node-init.sh, which installs some dependencies required for Mizar for the first-time deployment.
We will double-check this deployment on the AWS cloud as well.

Reply:
I am primarily using the currently checked in yaml (without PR https://github.com/CentaurusInfra/mizar/pull/513 merged yet) and it takes on average about 30s for the Mizar networking to be fully up and running.

Let's get to the bottom of why this is different for Arktos before introducing a separate path for arktos.

Reply:
When we deploy arktos on a fresh ubuntu machine according to [arktos guide](https://github.com/CentaurusInfra/arktos#build-arktos), and kernel updated to 5.6.
No mizar related stuff is installed prior such as python3.7 and python packages, everything will be install by node-init script.
It takes time for first-time deployment only.

If we install all the dependencies prior then mizar deployed in few seconds.

Reply:
In that case, mizar bootstrap script should be run to install kernel 5.6 and packages, user guide should be updated to run the script, and arktos-up should detect if the kernel version is not at the desired level, print a help message to run mizar bootstrap script before running arktos-up.

And we should use yaml for deployment of mizar components and bring up mizar networking during arktos-up.

Reply:
As per your suggestion, we have added a guide for deploying Arktos with Mizar.
Now, we can use YAML for Mizar component deployment.